### PR TITLE
Build fixes

### LINKED ISSUE #181: update versioning script for assets (/bin/version-assets.js)
I started this task and then realized after looking at code that we need to de-dupe `/server/page.js` and `client/js/pages/landing/index.js`, since they are both providing the page container (html tag, head tag with css refs and scripts in footer). prefer that this be done server side vs. client side, so i'll probably move `server/page.js` to `server/layouts/app.js` and have that be the place where the asset versioning script looks for assets.

### PR DESCRIPTION
**Pull request by laurengarcia**
some updates/fixes here for production:

* fix broken build caused by removing the browserify transform here: https://github.com/datproject/dat.land/commit/fd2b004fa90b39697d2687ea08f32645557340b7
* now that the browserify build transform has been brought back, it needed to be switched from using `es2020` to `es2040` to support object destructuring syntax
* the css minifier was also borked up by a comment i added that was explaining how the scss magic importer worked (why was it looking at commented-out code tho?)
* closes #181 (updated the minification versioning script and set up a ?debug=true path in the router that returns the unminified resources for debugging)

### CODE REVIEW

**Comment by karissa** [comment]
nice!

File: package.json [hunk 1]
```diff
@@ -44,8 +44,8 @@
     "chromedriver": "^2.21.2",
     "dat-design": "^1.4.1",
     "drag-drop": "^2.11.0",
-    "minidux": "^1.0.1",
```
Comment (karissa) [inline package.json]: heheh nice catch

**Comment by laurengarcia** [comment]
cool -- im going to merge this so we can start deploying to uat.

File: package.json (hunk 1 above)
Comment (laurengarcia) [inline package.json]: i kind of miss minidux tho :')

**Pull request merged**